src/domain/permissions.js

```diff
@@ -1,6 +1,6 @@
 import { ROLE } from './constants.js';
 
-const VIEW_USERS = 'view-users';
+const BROWSE_USERS = 'browse-users';
```

Comment:
Technically, we could have deleted this permission completely, because the only place it is used now, is equivalent with `MANAGE_ASSIGNED_EDITORS`, I re-introduced it after deleting it under the new name `BROWSE_USERS` as it makes more sense to protect the user search endpoint with it.